### PR TITLE
chore: Minor Tasks for Auto Layout feature, Sentry errors, metrics and Beta tag

### LINKED ISSUE #19712: [Chore]: Add Beta tag for Auto Layout V1
Add the following text to the Convert to AutoLayout button: (Beta)

New label - Convert to AutoLayout (beta)

### PR DESCRIPTION
**Pull request by rahulramesha**
## Description

This PR includes changes for,
- Adding common error messages for sentry issues when user decides to send error message, to enable searching of the reported issues on sentry
- Adding Analytics for conversion modal flow, to track user adoption of the feature
- Add Beta tag to the conversion button

Fixes #19712 

Media
image

## Type of change

- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?
- Manual

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/sagas/SnapshotSagas.ts [hunk 1]
```diff
@@ -117,13 +120,23 @@ function* restoreApplicationFromSnapshotSaga() {
...
+
     log.error(error);
     //update conversion form state to error
     yield put(
-      setLayoutConversionStateAction(CONVERSION_STATES.COMPLETED_ERROR),
+      setLayoutConversionStateAction(CONVERSION_STATES.COMPLETED_ERROR, error),
     );
-    throw error;
+
+    AnalyticsUtil.logEvent("CONVERSION_FAILURE", {
```
Comment (marks0351) [inline app/client/src/sagas/SnapshotSagas.ts:137]: Can we add appid as well to all logEvents to assess how many unique apps were converted.

**Review by marks0351 — DISMISSED** [review]

**Review by prsidhu — APPROVED** [review]

**Pull request merged**